docs/UserGuide.md

```diff
@@ -243,25 +243,34 @@ Examples:<br>`settle 1 amount/500.00`
 
 Deletes the specified student from the address book.
 
-Format: `delete INDEX`
+**Format:** `delete INDEX`
 
-* Deletes the student at the specified `INDEX`.
-* The index refers to the index number shown in the displayed student list.
+**Examples:**
+* `list` followed by `delete 2` deletes the 2nd student in the address book.
```

Comment:
I am not sure if we need to change "address book" to "UG Teach"

Reply:
Thanks for catching this! Ive changed it to "stored in UGTeach" instead of "the addressbook" to reduce ambiguity.